Missing regex include in `plugins/AccessLogger.h`

**Describe the bug**

```
/root/.conan2/p/drogo3ff2050bef073/p/include/drogon/plugins/AccessLogger.h:125:10: error: ‘regex’ in namespace ‘std’ does not name a type
  125 |     std::regex exemptRegex_;
```

**To Reproduce**

```
$ cat conanfile.txt
[requires]
drogon/1.9.12

[generators]
PkgConfigDeps
MesonToolchain
```

```dockerfile
$ cat Dockerfile
FROM ubuntu:24.04 AS builder

RUN apt update \
    && apt upgrade --yes \
    && apt install --yes \
        g++ \
        gcc \
        meson \
        pkg-config \
        python3 \
        python3-venv

WORKDIR /app

RUN python3 -m venv venv
RUN ./venv/bin/pip install conan==2.26.0
RUN ./venv/bin/conan profile detect

COPY conanfile.txt .
RUN ./venv/bin/conan install . \
    --output-folder build \
    --build missing \
    --settings build_type=Release \
    --settings compiler.cppstd=23 \
    --conf tools.system.package_manager:mode=install

COPY meson.build main.cpp .
RUN cd build && meson setup --native-file conan_meson_native.ini ..
RUN meson compile -C build

FROM ubuntu:24.04 AS runner
WORKDIR /app
RUN apt update && apt upgrade --yes && apt install --yes curl
COPY drogon.json .
COPY --from=builder /app/build/a .
ENTRYPOINT ./a
```

```json
$ cat drogon.json
{
  "listeners": [
    {
      "address": "127.0.0.1",
      "port": 8080
    }
  ],
  "app": {
    "log": {
      "log_level": "WARN"
    },
    "server_header_field": "drogon",
    "upload_path":"./drogon_uploads"
  },
  "plugins": [
    {
      "name": "drogon::plugin::AccessLogger",
      "config": {
        "log_format": "$request_date $remote_addr $method $url $status $body_bytes_sent $processing_time"
      }
    }
  ]
}
```

```meson
$ cat meson.build
project( 'mre', ['cpp'])
drogon_dep = dependency('drogon', include_type: 'system')
executable('a',  [ 'main.cpp' ], dependencies: [drogon_dep])
```

```cpp
$ cat main.cpp
#include <exception>
#include <iostream>
#include <numeric>
#include <ranges>
#include <vector>

// #include <drogon/drogon.h>
#include <drogon/plugins/AccessLogger.h>
#include <drogon/HttpAppFramework.h>

std::string get_drogon_addrs() {
  auto ret = std::vector<std::string>();
  std::ranges::transform(drogon::app().getListeners(), std::back_inserter(ret),
                         [](const auto &x) { return x.toIpPort(); });
  return "[" +
         std::accumulate(
             ret.begin(), ret.end(), std::string(),
             [](const auto &a, const auto &b) { return a + " " + b; }) +
         "]";
}

int main() try {
  using namespace drogon;

  app()
      .loadConfigFile("drogon.json")
      .registerHandler(
          "/",
          [](const HttpRequestPtr &request,
             std::function<void(const HttpResponsePtr &)> &&callback) {
            auto resp = HttpResponse::newHttpResponse();
            resp->setBody("Hello, World!\n");
            callback(resp);
          },
          {Get})
      .registerBeginningAdvice([] {
        std::clog << "Server listening " << get_drogon_addrs() << '\n';
      })

      .run();

  return 0;
} catch (const std::exception &ex) {
  std::cerr << "error: " << ex.what();
}
```

If I use drogon/drogon.h instead of separate headers it works.

Follow up question: is drogon supposed to be used only through drogon/drogon.h header or I'm allowed to use specific headers for separate classes?

Maybe related to #1561

Drogon 1.9.12
Ubuntu 24.04
gcc 13.3.0

--- a/lib/inc/drogon/plugins/AccessLogger.h
+++ b/lib/inc/drogon/plugins/AccessLogger.h
@@ -11,6 +11,7 @@
 #include <drogon/plugins/Plugin.h>
 #include <trantor/utils/AsyncFileLogger.h>
 #include <vector>
+#include <regex>
 
 namespace drogon
 {

